### PR TITLE
Prl 1569 respondent court needs

### PR DESCRIPTION
**Pull request by abdullahiafrahhmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/steps/respondent/safety_concerns/main_page/template.njk [hunk 1]
```diff
@@ -0,0 +1,46 @@
+
+{% extends "common/template.njk" %}
+
+{% block page_content %}
+
+
```
Comment (cshiwarale) [inline src/main/steps/respondent/safety_concerns/main_page/template.njk:6]: Please avoid unnecessary spacing

Comment (swaroopa86) [reply src/main/steps/respondent/safety_concerns/main_page/template.njk:6]: i will remove the additional spacing

File: src/main/app/case/CaseApi.ts [hunk 2]
```diff
@@ -165,6 +177,9 @@ export class CaseApi {
 }
 
 export const getCaseApi = (userDetails: UserDetails, logger: LoggerInstance): CaseApi => {
+  //console.log('s2stoken=====================>' + getServiceAuthToken());
+  //console.log('baseURL=====================>' + config.get('services.case.url'));
```
[the remark this answers is not in the document]
Comment (swaroopa86) [reply src/main/app/case/CaseApi.ts:181]: ok

File: src/main/app/case/CaseApi.ts [hunk 3]
```diff
@@ -144,7 +156,7 @@ export class CaseApi {
     //const data = toApiFormat(userData);
     //const data = userData;
     // return this.sendEvent(caseId, data, eventName);
-    console.log(eventName);
+    console.log('eventName = ' + eventName);
```
[the remark this answers is not in the document]
Comment (swaroopa86) [reply src/main/app/case/CaseApi.ts:159]: ok, but these changes are taken form master?

File: sonar-project.properties [hunk 4]
```diff
@@ -6,9 +6,9 @@ sonar.tests=src/test/
...
 sonar.javascript.lcov.reportPaths=coverage/lcov.info
-sonar.exclusions=src/**/*.test.ts,**/__mocks__/**,src/**/*content.ts,src/**/components/*.ts,src/**/definition.ts,src/**/summary/util.ts,src/**/summary/util.ts,src/**/steps/constants.ts,src/**/willingness-to-attend-miam/miam-cost-exemptions.ts,src/**/steps/index.ts,src/**/steps/urls.ts,src/**/case/CaseApi.ts,src/**/app/controller/PostController.ts,src/**/checkanswers/ConfirmContactDetailsGetController.ts,src/**/respondent/task-list/utils.ts,src/**/respondent/respondentcaseSequence.ts,src/main/steps/common/summary/utils.ts
+sonar.exclusions=src/**/*.test.ts,**/__mocks__/**,src/**/*content.ts,src/**/components/*.ts,src/**/definition.ts,src/**/summary/util.ts,src/**/summary/util.ts,src/**/steps/constants.ts,src/**/willingness-to-attend-miam/miam-cost-exemptions.ts,src/**/steps/index.ts,src/**/steps/urls.ts,src/**/case/CaseApi.ts,src/**/app/controller/PostController.ts,src/**/checkanswers/ConfirmContactDetailsGetController.ts,src/**/respondent/task-list/utils.ts,src/**/respondent/respondentcaseSequence.ts,src/main/steps/common/summary/utils.ts,src/**/applicant/task-list/utils.ts,src/**/ca-da-respondent/task-list/utils.ts
```
[the remark this answers is not in the document]
Comment (swaroopa86) [reply sonar-project.properties:9]: will remove this

File: src/main/app/case/CaseApi.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (cshiwarale) [reply src/main/app/case/CaseApi.ts:159]: If they are taken from master, then they should not be shown here as new addition, isn't it?

File: src/main/app/case/CaseApi.ts [hunk 5]
```diff
@@ -59,8 +59,11 @@ export class CaseApi {
   public async getCases(): Promise<CcdV1Response[]> {
     try {
       const response = await this.axios.get<CcdV1Response[]>(
-        `/citizens/${this.userDetails.id}/jurisdictions/${JURISDICTION}/case-types/${CASE_TYPE}/cases`
+        `http://ccd-data-store-api-aat.service.core-compute-aat.internal/caseworkers/${this.userDetails.id}/jurisdictions/${JURISDICTION}/case-types/${CASE_TYPE}/cases/1652796857708749`
```
[the remark this answers is not in the document]
Comment (cshiwarale) [reply src/main/app/case/CaseApi.ts:62]: If they are taken from master, then they should not be shown here as new addition, isn't it?

**Review by cshiwarale — APPROVED** [review]

**Pull request merged**